tools/circleci-upload-to-s3.sh

```diff
@@ -20,11 +20,18 @@ FILE_COUNT=$(find "${CT_REPORTS}" -type f | wc -l)
 echo "Uploading $FILE_COUNT files"
 ls $CT_REPORTS
 
+S3PP_COMMIT=6fd430a54e976d2d580042efdf82ac2fb66d5e57
+wget -O tools/s3-parallel-put https://raw.githubusercontent.com/arcusfelis/s3-parallel-put/$S3PP_COMMIT/s3-parallel-put
+chmod +x tools/s3-parallel-put
 
+sudo pip install boto python-magic
 sudo pip install awscli --upgrade --user
 aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID
 aws configure set aws_secret_access_key $AWS_SECRET_ACCESS_KEY
 aws configure set default.region $AWS_DEFAULT_REGION
-time aws s3 cp ${CT_REPORTS} s3://circleci-mim-results/${CT_REPORTS} --acl public-read --recursive --quiet
+
+sudo tools/s3-parallel-put --quiet --processes=64 --put=stupid \
```

Comment:
When I removed `sudo` the error occured:
```
  File "tools/s3-parallel-put", line 42, in <module>
    import magic #python-magic
ImportError: No module named magic
```

Reply:
That's probably happening because of the owner user, `python-magic` was installed with sudo, so if ran without sudo, the non-sudo user probably doesn't have it on his path. It wasn't happening for travis because we were running `sudo time pip install...`, where sudo applies to time and not to command. Can we do just `pip install...`?. I might be wrong with some detail here, but it seems to me to be a good lead.